code/modules/admin/admin.dm

```diff
@@ -862,4 +862,34 @@ proc/kick_clients_in_lobby(var/message, var/kick_only_afk = 0)
 				C << message
 			kicked_client_names.Add("[C.ckey]")
 			del(C)
-	return kicked_client_names
\ No newline at end of file
+	return kicked_client_names
+
+
+/datum/admins/proc/cmd_ghost_drag(var/mob/dead/observer/frommob, var/mob/living/tomob)
+
+	//this is the exact two check rights checks required to edit a ckey with vv.
+	if (!check_rights(R_VAREDIT,0) || !check_rights(R_SPAWN|R_DEBUG,0))
+		return 0
+
+	if (!frommob.ckey)
+		return 0
+
+	var/question = ""
+	if (tomob.ckey)
+		question = "This mob already has a user ([tomob.key]) in control of it! "
+	question += "Are you sure you want to place [frommob.name]([frommob.key]) in control of [tomob.name]?"
+
+	var/ask = alert(question, "Place ghost in control of mob?", "Yes", "No")
+	if (ask != "Yes")
+		return 1
+
+	tomob.ghostize(0)
+
+	message_admins("<span class='adminnotice'>[key_name_admin(usr)] has put [frommob.ckey] in control of [tomob.name].</span>")
+	log_admin("[key_name(usr)] stuffed [frommob.ckey] into [tomob.name].")
+	feedback_add_details("admin_verb","CGD")
+
+	tomob.ckey = frommob.ckey

```

Comment:
I thought about that, and the big issue is if the new mob is a special antag, like nuke ops or wizard, we want the ghost to keep the new mob's mind.

Another note, if that ghost was say, an apprentice, most cases we aren't gonna want the ghost to still have his mind spells.

I'll have it do it if the tomob has no mind.

This in general brings up the need to have better admin control over minds for cases like this